### PR TITLE
Add win11 support to `Get-WindowsIsoUrl`

### PR DESCRIPTION
**Pull request by sethbrasile**
Made `-Win` and `-Rel` params mandatory since we now need to choose between 10 and 11 and the `-Rel` param is not enough to differentiate because 10 and 11 share those designations.

Needs to be merged at the same time as PR for Win10 upgrade script: https://github.com/dkbrookie/Imaging/pull/4

### CODE REVIEW

File: Function.Get-WindowsIsoUrl.ps1 [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        Rel = "'$Rel'"
+    }
+
+    $fidoScriptBlock = [ScriptBlock]::create(".{$fidoScript} $(&{$args} @params)")
+
+
+    $url = Invoke-Command -ScriptBlock $fidoScriptBlock
+
+    If ($Null -eq $url) {
+        Throw "FIDO (WinISO URL generator script) is not returning a value!! Must fix FIDO!"
```
Comment (wmmatt) [inline Function.Get-WindowsIsoUrl.ps1:54]: Isn't throw usually used in a try/catch block? It's fine to just use it in the open like this?

File: Function.Get-WindowsIsoUrl.ps1 [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+function Get-WindowsIsoUrl {
+    param(
+        [Parameter(Mandatory)]
+        [ValidateSet('20H2', '21H1', '21H2')]
+        [string]$Rel,
+        [Parameter(Mandatory)]
+        [ValidateSet(10, 11)]
+        [string]$Win
+    )
```
Comment (wmmatt) [inline Function.Get-WindowsIsoUrl.ps1:37]: Agreed on the mandatory, read yoru other notes on this too. Looks good.

**Review by wmmatt — COMMENTED** [review]
Had a question on your Throw if you can take a look real quick. Googling usage of it too in the mean time.

**Comment by wmmatt** [comment]
> The expression in the Throw syntax is optional. When the Throw statement does not appear in a Catch block, and it does not include an expression, it generates a ScriptHalted error.

Ah there we go...so it stops the script on error. 10-4. Sorry had just never seen it outside of try/catch block before and it confused me hah

**Comment by sethbrasile** [comment]
Yep @wmmatt and you can catch the `throw` in a try/catch wherever you call this function as well

**Review by wmmatt — APPROVED** [review]
Looks good

**Pull request merged**